bot.ts

```diff
@@ -36,90 +57,158 @@ const commands: SlashCommandPartial[] = [
-		await t.defer();
+	async seriesCommunity(i: ApplicationCommandInteraction) {
+		await i.defer();
 		const res = await fetch(
-			`https://www.speedrun.com/api/v1/series/${t.option("series")}`,
+			`https://www.speedrun.com/api/v1/series/${i.option("series")}`,
 		);
-		const series = (await res.json()).data as SearchResult;
-
-		if (series) {
-			await t.reply({
-				content: series.discord
-					? `Here's the invite to the \`${series.names.international} series\`: ${series.discord}`
-					: `Couldn't find any invite for the \`${series.names.international} series\`, here's a link to their forums: ${series.weblink}/forum`,
-			});
-		} else {
-			await t.reply({
-				content: `\`${t.option("series")} series\` not found`,
-			});
+		const series: v1SearchResult | undefined = (await res.json()).data;
+		const userMention = i.option("mention");
+	
+		if (!series) {
+			await i.reply(
+				userMention
+					? `${userMention}, the \`${i.option("series")} Series\` wasn't found`
+					: `\`${i.option("series")} Series\` not found`
+			);
+			return;
+		}
+
+		// series.weblink doesn't have the "/series/" prefix
+		const formLink = `https://www.speedrun.com/series/${series.abbreviation}/forums`;
+
+		if (userMention) {
+			await i.reply(
+				series.discord
+					? `${userMention}, here's the invite to the \`${series.names.international} Series\`: ${series.discord}`
+					: `${userMention}, couldn't find any Discord invite for the \`${series.names.international} Series\`. Here's a link to their forums: ${formLink}`,
+			);
+			return;
 		}
+
+		await i.reply(
+			series.discord
+				? `Here's the invite to the \`${series.names.international} Series\`: ${series.discord}`
+				: `Couldn't find any Discord invite for the \`${series.names.international} Series\`. Here's a link to their forums: ${formLink}`,
+		);
+	}
+
+	@slash()
+	async github(i: ApplicationCommandInteraction) {
+		await i.reply("Here's the link to the GitHub repository: https://github.com/AnInternetTroll/speedruncom-discord-bot");
 	}
 
 	@autocomplete("*", "*")
-	async Autocomplete(d: AutocompleteInteraction) {
-		const completions: ApplicationCommandChoice[] = [];
-		const isGame = Boolean(d.option("game"));
-		const res = await fetch(
+	async myAutocomplete(d: AutocompleteInteraction) {
+		if (!d.focusedOption.value) {
+			await d.autocomplete([]);
+			return;
+		}
+
+		const isGame = Boolean(d.focusedOption.name == "game");
+
+		const v2Res = await fetch(`https://www.speedrun.com/api/v2/GetSearch?_r=${
+			btoa(JSON.stringify({
+				query: d.focusedOption.value,
+				limit: 20,
+				favorExactMatches: false,
+				includeGames: isGame,
+				includeSeries: !isGame
+			})).replace(/=+$/, "")
+		}`);
+
+		if (v2Res.ok) {
+			const body: v2SearchResult = await v2Res.json();
+
+			const list = isGame ? body.gameList : body.seriesList;
+			await d.autocomplete(
+				list.map((x) => ({
+					name: isGame ? x.name : `${x.name} Series`,
```

Comment:
Isn't "Series" redundant here? Since it's the series parameter you are inputting

Reply:
I don't think so, since this was also in the version before. And that historically the word Series has been after the series' name.